HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift

```diff
@@ -150,22 +153,22 @@ final class InvoicesListViewModel {
                                                 metadata: nil) { [weak self] result in
                 switch result {
                 case .success(let createdDocument):
-                    Log("Successfully created document with id: \(createdDocument.id)", event: .success)
+                    print("Successfully created document with id: \(createdDocument.id)")
```

Comment:
[Suggestion]: @razvancapra please find alternative for `print`, we can't keep it in production

Reply:
@zladzeyka  `Log` function is inside `GiniHealthAPILibrary` and it's used there. Should I copy/paste it in the GiniUtilities and use it from there to the clients? Will we expose GiniUtilities to the clients as well?

Reply:
yes please